e2e/setup-e2e.sh

```
@@ -313,16 +305,24 @@ setup_test_environment() {
+    # failures propagate under set -e (local VAR=$(...) masks exit codes).
+    local BASEDOMAIN
     if [ "$METHOD" = "operator" ]; then
         # For operator deployment, construct the endpoint from the Jumpstarter CR
         # The operator uses nodeport mode by default with port 8082
-        local BASEDOMAIN=$(kubectl get jumpstarter -n "${JS_NAMESPACE}" jumpstarter -o jsonpath='{.spec.baseDomain}')
+        BASEDOMAIN=$(kubectl get jumpstarter -n "${JS_NAMESPACE}" jumpstarter -o jsonpath='{.spec.baseDomain}')
         export ENDPOINT="grpc.${BASEDOMAIN}:8082"
+        export LOGIN_ENDPOINT="login.${BASEDOMAIN}:8086"
     else
         # For helm deployment, get the endpoint from helm values
         export ENDPOINT=$(helm get values jumpstarter --output json | jq -r '."jumpstarter-controller".grpc.endpoint')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)